pact/ledger/ledger.pact

```diff
@@ -213,36 +213,40 @@
       s)
   )
 
-  (defun create-token-id:string (token-details:object{token-details})
+  (defun create-token-id:string (token-details:object{token-details}
+                                 creation-guard:guard)
     (format "t:{}"
-      [(hash [token-details (at 'chain-id (chain-data))])])
+      [(hash [token-details (at 'chain-id (chain-data)) creation-guard])])
   )
 
   (defun create-token:bool
     ( id:string
       precision:integer
       uri:string
       policies:[module{kip.token-policy-v2}]
+      creation-guard:guard
     )
     ;; enforces token and uri protocols
     (enforce-uri-reserved uri)
     (let ((token-details { 'uri: uri, 'precision: precision, 'policies: (sort policies) }))
-      (enforce-token-reserved id token-details)
-    )
-    (with-capability (INIT-CALL id precision uri)
-      ;; maps policy list and calls policy::enforce-init
-      (marmalade-v2.policy-manager.enforce-init
-        { 'id: id, 'supply: 0.0, 'precision: precision, 'uri: uri,  'policies: policies})
+      (enforce-token-reserved id token-details creation-guard)
     )
+    (with-capability (TOKEN id precision policies uri creation-guard)
```

Comment:
I don't believe we need to pass in creation signature to `enforce-init`. Otherwise, all the policies will have sig from creator-guard.  It makes more sense to put in L#240, like the other function structures.